webextension/scripts/tagcloud.js

```diff
@@ -17,7 +17,7 @@ function getTags() {
   let y = hostname.split('.')
   let not_display4 = y.join(' ')
   let not_display1 = y.join(' ')
-  $('.url').text(url).attr('href', url)
+  $('.url').text(hostname).attr('href', hostname)
```

Comment:
If you click on the link, it fails due to missing `http(s)://`. Could keep the href pointed to `url`